configurator/frontend/src/lib/components/OnboardingTour/steps/OnboardingTourAddDestination/OnboardingTourAddDestination.module.less

```diff
@@ -0,0 +1,41 @@
+@import "../commons/OnboardingCommonStyles.module.less";
+
+.mainContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  max-height: 70vh;
+  max-width: 100%;
+  height: 100vh;
+  width: 100vw;
+}
+
+.header {
+  font-size: @header-font-size;
+  font-weight: @header-font-weight;
+  text-align: center;
+  margin-bottom: 20px;
+}
+
+.paragraph {
+  font-size: @text-font-size;
+  margin-bottom: 20px;
+}
+
+.addDestinationButtonContainer {
+  margin-bottom: 10px;
+}
+
+.destinationEditorContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  flex: 1 1 100%;
+  width: 100%;
+}
```

Comment:
I'd rather use tailwind classes here via `@apply`. For consistency reasons

Reply:
Sure, will do. Thanks for pointing to that!